recipes/swig/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "3.0.10" %}
+
+package:
+  name: swig
+  version: {{ version }}
+
+source:
+  fn: swig-{{ version }}.tar.gz                                             # [unix]
+  url: http://prdownloads.sourceforge.net/swig/swig-{{ version }}.tar.gz    # [unix]
+  sha256: 2939aae39dec06095462f1b95ce1c958ac80d07b926e48871046d17c0094f44c  # [unix]
+  fn: swigwin-{{ version }}.zip                                             # [win]
+  url: http://prdownloads.sourceforge.net/swig/swigwin-{{ version }}.zip    # [win]
+  sha256: 68a202ebfc62647495074a190a115b629e84c56d74d3017ccb43e56a4b9b83f6  # [win]
+
+build:
+  number: 0
+  skip: true                             # [not py27]
```

Comment:
So, I picked one version of `python`, but am not sure if we should be using more and making `python` a run-time dependency or not.